src/htmlparser.js

```diff
@@ -294,15 +294,29 @@ function HTMLParser(html, handler) {
         if (args[5] === '') { args[5] = undefined; }
       }
 
+      function populate(index) {
+        customAssign = args[index];
+        value = args[index + 1];
+        if (value !== undefined) {
```

Comment:
Hm, I think the convention we follow with `undefined` checks is to via `typeof` (or just `!value` if it can never be falsy at that point)